src/dbnode/persist/fs/commitlog/commit_log.go

```diff
@@ -238,6 +235,31 @@ func (r callbackResult) rotateLogsResult() (rotateLogsResult, error) {
 	return r.rotateLogs, nil
 }
 
+func zeroToTenSecondsHighResDurationBuckets() tally.Buckets {
```

Comment:
This seems like a lot of buckets, any concerns for cardinality / instrumentation perf here?

Reply:
Hm actually looks like the other buckets we have is ~60 so this is probably fine